css/style.css

```diff
@@ -902,6 +910,12 @@ button.button-inline:hover {
 	}
 }
 
+@media (max-width: 768px) {
+	#app-navigation-toggle {
+		display: none !important;
```

Comment:
@juliushaertl didn't you mention somewhere that navigation toggle should be available on mobile as well?